misc/msr-lock.md

```diff
@@ -25,60 +23,96 @@ So to fix it we have 2 options:
+* [UEFITool](https://github.com/LongSoft/UEFITool/releases) (Make sure it's UEFITool and not UEFIExtract)
 * [Universal-IFR-Extractor](https://github.com/LongSoft/Universal-IFR-Extractor/releases)
 
-And don't forget to disable the following from your config.plist under `Kernel -> Quirks`:
-
-* `AppleCpuPmCfgLock`
-* `AppleXcpmCfgLock`
-
 And the final part, grabbing your BIOS from the vendors' website.
 
 Now the fun part!
 
-## Checking if CFG-Lock can be turned off
 
-Boot OpenCore and select the `VerifyMsrE2` option in the picker. This tool will tell you whether your BIOS supports CFG-Lock and if it can be unlocked.
 
 ## Turning off CFG-Lock manually
 
+##### Please note that the only firmwares that can be directly opened by UEFITool are ASUS, MSI and ASRock. Other firmwares need a special procedure which we'll not directly cover into this guide. For Dell firmwares please refer to [dreamwhite's guide](https://github.com/dreamwhite/bios-extraction-guide/tree/master/Dell)
+
 1. Open your firmware with UEFITool and then find `CFG Lock` as a Unicode string. If nothing pops up then your firmware doesn't support `CFG Lock`, otherwise continue on.
 
 ![](../images/extras/msr-lock-md/uefi-tool.png)
 
-1. You'll find that this string is found within a Setup folder, right-click and export as `Setup.bin`
+1. You'll find that this string is found within a Setup folder, right-click and export as `Setup.bin` (or even `Setup.sct`)
 2. Open your setup file with `ifrextract` and export as a .txt file with terminal:
 
    ```
    path/to/ifrextract path/to/Setup.bin path/to/Setup.txt
    ```
 
-3. Open the text file and search for `CFG Lock, VarStoreInfo (VarOffset/VarName):` and note the offset right after it(ie: `0x5A4`)
+3. Open the text file and search for `CFG Lock, VarStoreInfo (VarOffset/VarName):` and note the offset right after it (ie: `0x5A4`)
 
 ![](../images/extras/msr-lock-md/cfg-find.png)
 
-1. Run the Modified GRUB Shell and paste the following where `0x5A4` is replaced with your value:
+1. Run the Modified GRUB Shell and write the following command where `0x5A4` is replaced with the value you've previously extracted:
 
    ```
-   setup_var 0x5A4 0x00
+   setup_var 0x5A4
    ```
 
+   If you get an error such as `error: offset is out of range` run the following command:
+   
+   ```
+   setup_var2 0x5A4
+   ```
+   
+   Just as before, if you still get `error: offset is out of range` you'd need to use this command:
+   
+   ```
+   setup_var_3 0x5A4
+   ```
+   
+   If you don't get any type of error, write the command which doesn't lead to `error: offset is out of range` (e.g. `setup_var_3 0x5A4`) and write `0x00` after it:
+   
+   ```
+   setup_var_3 0x5A4 0x00
+   ```
+   
+   At this point **turn off** the PC then start it again and you'll have `CFG Lock` unlocked.
    Do note that variable offsets are unique not just to each motherboard but even to its firmware version. **Never try to use an offset without checking.**
```

Comment:
With markdown, this will appear as 1 paragraph. I think it would be a little cleaner to have them separate